mne/decoding/transformer.py

```diff
@@ -113,12 +113,15 @@ def __init__(self, info=None, scalings=None, with_mean=True,
 
         if not (scalings is None or isinstance(scalings, (dict, str))):
             raise ValueError('scalings type should be dict, str, or None, '
-                             'got %s' % type(scalings))
+                             'got {}'.format(type(scalings)))
         if isinstance(scalings, string_types) and \
                 scalings not in ('mean', 'median'):
             raise ValueError('Invalid method for scaling, must be "mean" or '
-                             '"median" but got %s' % scalings)
+                             '"median" but got {}'.format(scalings))
```

Comment:
I still think you should format all 3 the same way ... otherwise it's ready I'd say.